docs/submission/index.md

```diff
@@ -3,70 +3,107 @@ hide:
   - toc
 ---
 
-If you follow the `cm run` commands under the individual model pages in the [benchmarks](../benchmarks/index.md) directory, all the valid results will get aggregated to the `cm cache` folder. Once all the results across all the modelsare ready you can use the following command to generate a valid submission tree compliant with the [MLPerf requirements](https://github.com/mlcommons/policies/blob/master/submission_rules.adoc#inference-1).
+=== "CM based benchmark"
+    If you have followed the `cm run` commands under the individual model pages in the [benchmarks](../index.md) directory, all the valid results will get aggregated to the `cm cache` folder. The following command could be used to browse the structure of inference results folder generated by CM.
+    ### Get results folder structure
+    ```bash
+    cm find cache --tags=get,mlperf,inference,results,dir | xargs tree
```

Comment:
Should we create a new script in CM for the purpose of showing the directory structure using tree?

Reply:
@anandhu-eng what's the advantage?

Reply:
We could prevent the usage of commands other than CM in documentation. I don't find any specific advantage.

Reply:
Thats not necessary. `tree` command is fine to be used.